src/main/kotlin/apifi/codegen/Non200ResponseHandler.kt

```diff
@@ -0,0 +1,30 @@
+package apifi.codegen
+
+import org.apache.http.HttpStatus
+
+object Non200ResponseHandler {
+
+    private val allExceptionDetailsHolder = listOf<ExceptionDetailsHolder>(
```

Comment:
I could not think of better approach, please suggest alternate for this class if any.

Reply:
ErrorResponseHandler??